webapp/allcards.js

```diff
@@ -339,7 +339,68 @@ allcards_data=
 {"id":"Tower","box":"empires","type":"Landmark","event":true},
 {"id":"Triumphal Arch","box":"empires","type":"Landmark","event":true},
 {"id":"Wall","box":"empires","type":"Landmark","event":true},
-{"id":"Wolf Den","box":"empires","type":"Landmark","event":true}//,
+{"id":"Wolf Den","box":"empires","type":"Landmark","event":true},
+{"id":"Bard","box":"nocturne","type":"Action,Fate","cost":"4"},
+{"id":"Blessed Village","box":"nocturne","type":"Action,Fate","cost":"4"},
+{"id":"Changeling","box":"nocturne","type":"Night","cost":"3"},
+{"id":"Cobbler","box":"nocturne","type":"Night,Duration","cost":"4"},
+{"id":"Conclave","box":"nocturne","type":"Action","cost":"4"},
+{"id":"Crypt","box":"nocturne","type":"Night,Duration","cost":"5"},
+{"id":"Cursed Village","box":"nocturne","type":"Action,Doom","cost":"5"},
+{"id":"Den of Sin","box":"nocturne","type":"Night,Duration","cost":"5"},
+{"id":"Devil's Workshop","box":"nocturne","type":"Night","cost":"4"},
+{"id":"Druid","box":"nocturne","type":"Action,Fate","cost":"2"},
+{"id":"Exorcist","box":"nocturne","type":"Night","cost":"4","requires":["Ghost","Imp","Will-o'-Wisp"]},
+{"id":"Faithful Hound","box":"nocturne","type":"Action,Reaction","cost":"2"},
+{"id":"Fool","box":"nocturne","type":"Action,Fate","cost":"3","requires":["Lucky Coin","States"]},
+{"id":"Ghost Town","box":"nocturne","type":"Night,Duration","cost":"3"},
+{"id":"Guardian","box":"nocturne","type":"Night,Duration","cost":"2"},
+{"id":"Idol","box":"nocturne","type":"Treasure,Attack,Fate","cost":"5"},
```

Comment:
where needed, it would be nice to identify Boons and/or Hexes being required.
E.g. when "Idol" is in play, Boons are needed.

Reply:
I agree that it could be more obvious.

Cards with type "Fate" use Boons, and cards with type "Doom" use Hexes.

Initially I used the "requires" field to specify the Boons/Hexes, but I changed it to match the behaviour of Looter cards requiring Ruins in Dark Ages.